go.sum

```diff
@@ -1,5 +1,18 @@
 github.com/BurntSushi/toml v0.3.1/go.mod h1:xHWCNGjB5oqiDr8zfno3MHue2Ht5sIBksp03qcyfWMU=
+github.com/alecthomas/gometalinter v2.0.11+incompatible h1:ENdXMllZNSVDTJUUVIzBW9CSEpntTrQa76iRsEFLX/M=
```

Comment:
Did all these new deps come from the prompt lib?

Reply:
Correct https://github.com/manifoldco/promptui/blob/master/go.mod#L4